src/js/Aladin.js

```diff
@@ -1272,7 +1277,25 @@ export let Aladin = (function () {
     };
 
     // Select corresponds to rectangular selection
-    Aladin.AVAILABLE_CALLBACKS = ['select', 'objectClicked', 'objectHovered', 'footprintClicked', 'footprintHovered', 'positionChanged', 'zoomChanged', 'click', 'mouseMove', 'fullScreenToggled', 'catalogReady'];
+    Aladin.AVAILABLE_CALLBACKS = [
+        'select',
+
+        'objectClicked',
+        'objectHovered',
+        'objectHoveredStop',
+
+        'footprintClicked',
+        'footprintHovered',
+
+        'positionChanged',
+        'zoomChanged',
+
+        'click',
+        'mouseMove',
+
+        'fullScreenToggled',
+        'cooFrameChanged'
```

Comment:
Should the right click be there too?

Reply:
Yes, I see a usecase but when the user is rightclicking while moving the mouse (this by default change the cut on the selected layer but it could do what the user defines).

Should a rightclickMove would be appropriate ? If AVAILABLE_CALLBACKS['rightclickMove'] is defined, call it instead of changing the cuts.